packages/merge/src/components/MergeSummary/MergeSummary.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Typography} from 'tocco-ui'
+import {intlShape} from 'react-intl'
+import _groupBy from 'lodash/groupBy'
+
+const MergeSummary = ({mergeResponse, intl}) => {
+  const msg = id => intl.formatMessage({id})
+
+  const notCopiedRelations = Object.entries(_groupBy(mergeResponse.body.notCopiedRelations, e => {
```

Comment:
i wouldnt add "body" to the store but the object containing "notCopiedRelations" directly. Body is a http request thing and a component should know nothing about it